<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodEditDialog</class>
 <widget class="QDialog" name="GoodEditDialog">
  <property name="minimumWidth">
   <number>420</number>
  </property>
  <property name="windowTitle">
   <string>Товар</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="maxLength">
        <number>200</number>
       </property>
       <property name="placeholderText">
        <string>Название товара</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="descriptionEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Описание (необязательно)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="categoryCombo">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="activeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="activeCheck">
       <property name="text">
        <string>Активен</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string>Пиктограмма:</string>
       </property>
       <property name="toolTip">
        <string>Изображение для карточки товара в поиске (PNG, JPG)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QWidget" name="imageWidget" native="true">
       <layout class="QHBoxLayout" name="imageLayout">
        <item>
         <widget class="QLabel" name="iconPreviewLabel">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string>—</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imageUrlEdit">
          <property name="placeholderText">
           <string>Путь к файлу или пусто</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseImageButton">
          <property name="text">
           <string>Обзор…</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
